commit da75e5130d8e1705597423ddcb1e9b2ed8fefe18
Author: Pedro Palazon Candel <pedro@joyent.com>
Date:   2019-06-17T16:17:09+02:00 (4 months ago)
    
    TRITON-1741 Flexible Disk cannot be sized down
    Reviewed by: Mike Gerdts <mike.gerdts@joyent.com>
    Approved by: Mike Gerdts <mike.gerdts@joyent.com>

diff --git a/lib/tritonapi.js b/lib/tritonapi.js
index dfa3918..a3406e0 100644
--- a/lib/tritonapi.js
+++ b/lib/tritonapi.js
@@ -4107,7 +4107,7 @@ function resizeInstanceDisk(opts, cb) {
                 id: arg.instId,
                 diskId: arg.diskId,
                 size: opts.size,
-                dangerousAllowSrhink: opts.dangerousAllowShrink
+                dangerousAllowShrink: opts.dangerousAllowShrink
             }, function (err, _, _res) {
                 res = _res;
                 res.instId = arg.instId; // gross hack, in case caller needs it
